var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -37,17 +40,20 @@ class PyPytest(PythonPackage):
     depends_on('py-setuptools@30.3:', when='@3.9.0:3.9.1', type=('build', 'run'))
     depends_on('py-setuptools', type=('build', 'run'))
     depends_on('py-setuptools-scm', when='@3.1:', type='build')
-    depends_on('py-py@1.5.0:', when='@3.3:', type=('build', 'run'))
+    depends_on('py-py@1.8.2:', when='@6:', type=('build', 'run'))
+    depends_on('py-py@1.5.0:', when='@3.3:5.999', type=('build', 'run'))
     depends_on('py-py@1.4.33:', when='@3.1.2:3.2.3,3.2.5:3.2.999', type=('build', 'run'))
     depends_on('py-py@1.4.33:1.4.999', when='@3.2.4', type=('build', 'run'))
     depends_on('py-py@1.4.29:', when='@:3.1.1', type=('build', 'run'))
     depends_on('py-six@1.10.0:', when='@3.3:4', type=('build', 'run'))
     depends_on('py-packaging', when='@4.6:', type=('build', 'run'))
+    depends_on('py-attrs@19.2.0:', when='@6.1.1:', type=('build', 'run'))
     depends_on('py-attrs@17.2.0:', when='@3.3:3.4', type=('build', 'run'))
     depends_on('py-attrs@17.4.0:', when='@3.5:', type=('build', 'run'))
     depends_on('py-more-itertools@4.0.0:', when='@3.5.1:', type=('build', 'run'))
```

Comment:
This dep isn't needed in the latest version

Reply:
I kept the line because in `pytest-6.2.2/setup.cfg` file, there is still a line 
```
attrs>=19.2.0
```

I modified the *when* for `py-attrs@19.20` et `py-attrs@17.4.0`

Reply:
I'm commenting on more-itertools, not attrs

Reply:
sorry, I was focus on the modified line.